chainer/functions/utils/forget.py

```diff
@@ -59,4 +59,59 @@ def backward(self, inputs, grads):
 
 
 def forget(func, *xs):
+    """Call a function without storing internal results.
+
+    On a forward-propagation Chainer stores all internal results of
+    :class:`Function` on a computational graph as they are required on
+    backward-propagation. These results consume too many memories. This method
```

Comment:
I still think this sentence alone can be read that they __always__ consume too many memories even when the model is small because some procedure is doing some inefficient things. This is wrong. How about writing this sentence as """These results consume too many memories when the internal results are too large.""""?